allfields.md

```diff
@@ -0,0 +1,167 @@
+## LeadIQ Contact Search - All Fields
+
+```
+query SearchPeople($input: SearchPeopleInput!) {
+  searchPeople(input: $input) {
+    totalResults
+    hasMore
+    results {
+      _id
+      name {
+        first
+        middle
+        last
+      }
+      personalEmails {
+        type
+        status
+        sources {
+          source
+          status
+          providerIds
+        }
+        value
+        updatedAt
```

Comment:
I would suggest we split these up using fragments so its easier to see the common types.